Maple2.Database.Seed/Program.cs

```diff
@@ -52,18 +56,61 @@
 
 Console.WriteLine("Seeding complete!");
 
-
 void Seed(string type) {
     Stopwatch stopwatch = new();
 
-    Console.Write($"Seeding {type}... ");
+    Console.WriteLine($"Seeding {type}... ");
+
+    if (IsTableEmpty(type)) {
+        string fileLines = File.ReadAllText(Path.Combine(Paths.DB_SEEDS_DIR, $"{type}.sql"));
```

Comment:
this file read happens twice and its the same content, move this one above the if and reuse it below